src/boot.tsx

```diff
@@ -22,6 +22,22 @@ export default (element: HTMLDivElement, hub: Hub) => {
     applyMiddleware(createEpicMiddleware(rootEpic))
   );
 
+  const getPackages = (packages: string) => {
+    if (packages){
```

Comment:
I assume you want to check for an empty string given the type constraints.

Reply:
Here I wanted to check if `packages` is either `null` (in case packages are not defined in the html at all) or the empty string (in case of packages defined but not specified, like `packages=""`). However, from my understanding `_.isNil(packages)` checks only the first of the two cases, am I correct?

Reply:
Is it possible to address this comment?